openfasoc/MLoptimization/gen_spec.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+## Generate the design specifications and then save to a pickle file
+
+import numpy as np
+import random
+import yaml
+import os
+import argparse
+
+def gen_data(env, num_specs):
+
+  specs_range = {
+                "gain_min" : [float(1000338000.0), float(3000338000.0)],
+                "FOM" : [float(5*10**11), float(5*10**11)]
+                }
```

Comment:
Add variables for each of these constants, and provide descriptive names for each.